src/main/java/org/terasology/computer/display/component/DisplayComponent.java

```diff
@@ -62,4 +51,12 @@ public void setMode(String mode) {
     public void setData(List<String> data) {
         this.data = data;
     }
+
+    @Override
+    public void copy(DisplayComponent other) {
+        this.monitorSize = other.monitorSize;
+        this.front = other.front;
+        this.mode = other.mode;
+        this.data = Lists.newArrayList(data);
```

Comment:
If you change the constructor to use `addAll` , why not `.clear()` and `.addAll(other.data)` here? 

Also, there's a bug in there as it should at least be 
```suggestion
        this.data = Lists.newArrayList(other.data);
```

Reply:
like so?